packages/design-system/src/docs/SearchBar.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import algoliasearch from 'algoliasearch';
+import { getAlgoliaResults } from '@algolia/autocomplete-js';
+
+import { Autocomplete } from './AlgoliaAutocomplete';
+
+import theme from './SearchBar.scss';
+
+const searchClient = algoliasearch(
+	process.env.STORYBOOK_ALGOLIA_SEARCH_APP_ID || '',
+	process.env.STORYBOOK_ALGOLIA_SEARCH_API_KEY || '',
+);
+
+function SearchBar() {
+	if (!process.env.STORYBOOK_ALGOLIA_SEARCH_INDEX) {
+		console.warn(
+			'Algolia is not set up correctly, be sure to configure STORYBOOK_ALGOLIA_SEARCH_APP_ID, STORYBOOK_ALGOLIA_SEARCH_API_KEY and STORYBOOK_ALGOLIA_SEARCH_INDEX secrets before.',
+		);
+		return null;
+	}
+
+	return (
+		<div style={{ maxWidth: '50%', margin: '0 auto' }}>
+			<Autocomplete
+				/*
+                // @ts-ignore */
+				placeholder="Search for anything on Coral"
+				getSources={
+					/*
+                    // @ts-ignore */
```

Comment:
Weird double comment syntax, also missing the reason why we allow these ts-ignores